packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dateRangeBox.tests.js

```diff
@@ -1002,6 +1004,23 @@ QUnit.module('Behavior', moduleConfig, () => {
         assert.strictEqual(startDateBox.option('opened'), true, 'startDateBox is opened');
     });
 
+    QUnit.test('Cells in-range should be highlighted when pressing previous button in calendar (T1253076)', function(assert) {
```

Comment:
the test is evergreen

Reply:
[fbe8a22](https://github.com/DevExpress/DevExtreme/pull/28423/commits/fbe8a2281282c096308a8bf5bde0fa5f548498d5)